libraries/AP_Compass/AP_Compass.cpp

```diff
@@ -510,26 +501,26 @@ const AP_Param::GroupInfo Compass::var_info[] = {
     // @Values: 0:Disabled,1:CheckOnly,2:CheckAndFix
     AP_GROUPINFO("AUTO_ROT", 35, Compass, _rotate_auto, HAL_COMPASS_AUTO_ROT_DEFAULT),
 
-    // @Param: PRI_DEVID
-    // @DisplayName: Primary Compass device id
-    // @Description: Primary Compass device id, set automatically if 0. Reboot required after change.
+    // @Param: PRIO1_ID
```

Comment:
We should put a "@RebootRequired" on these parameters I think.. and maybe a few others?